# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/ModuleLoadRequest.cpp
# Commit: 9355004c7f6a
# Full Hash: 9355004c7f6a72ca7f8c11df9d70076626767ae9
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move mWebExtGlobal from ScriptLoadRequest to DOMScriptLoadContext; r=jonco,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D136744
# ==============================================================================

diff -r 0a5f61274aed -r 9355004c7f6a dom/script/ModuleLoadRequest.cpp
--- a/dom/script/ModuleLoadRequest.cpp	Thu Feb 17 15:57:48 2022 +0000
+++ b/dom/script/ModuleLoadRequest.cpp	Thu Feb 17 15:57:49 2022 +0000
@@ -163,7 +163,8 @@
 
   LOG(("ScriptLoadRequest (%p): Module loaded", this));
 
-  mModuleScript = mLoader->GetFetchedModule(mURI, GetWebExtGlobal());
+  mModuleScript =
+      mLoader->GetFetchedModule(mURI, GetLoadContext()->GetWebExtGlobal());
   if (!mModuleScript || mModuleScript->HasParseError()) {
     ModuleErrored();
     return;